modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -168,6 +169,22 @@ public void setUp() throws Exception {
 				_objectDefinition.getObjectDefinitionId()
 			).build());
 
+		ObjectFieldUtil.addCustomObjectField(
+			new LongIntegerObjectFieldBuilder(
+			).externalReferenceCode(
+				"Object_Definition_id__c"
+			).userId(
+				adminUser.getUserId()
+			).labelMap(
+				LocalizedMapUtil.getLocalizedMap("Object Definition Id")
+			).listTypeDefinitionId(
```

Comment:
I think we don't need to fill **listTypeDefinitionId** for LongInteger field type.